### ISSUE TITLE
Compiling type-unary-0.2.19 fails on GHC 7.6.3 / Ubuntu 12.04 / Cabal 1.22

### ISSUE THREAD

**Issue by edom** [conal/type-unary#2]
```
src/TypeUnary/Nat.hs:141:10:
    Variable `n' occurs more often than in the instance head
      in the constraint: (n :+: Z) ~ n
    (Use -XUndecidableInstances to permit this)
    In the instance declaration for `PlusZero n'
```

**Comment by conal** [comment]
Hm. With which compiler version?

**Comment by edom** [comment]
I compiled it with GHC 7.6.3.

**Comment by conal** [comment]
Oh! I see now that you included version info in the issue title. Fixed in commit 2eda7e9f4f6a316d69720e824ad648f1427d5fe1, and released as [type-unary-0.2.21](http://hackage.haskell.org/package/type-unary-0.2.21). Thanks for the report.

**Comment by edom** [comment]
Well, the title may be too long; I'll remember to write that info in the
body next time.

Thanks for the fix!
